src/main/fc/rc_controls.c

```diff
@@ -325,6 +325,8 @@ void processRcStickPositions(throttleStatus_e throttleStatus)
         cameraControlKeyPress(CAMERA_CONTROL_KEY_DOWN, 0);
     } else if (rcSticks == THR_LO + YAW_CE + PIT_HI + ROL_CE) {
         cameraControlKeyPress(CAMERA_CONTROL_KEY_UP, 2000);
+    }else if (rcSticks == THR_CE + YAW_HI + PIT_LO + ROL_CE) {                       //add enter key when it is in the camera osd
+        cameraControlKeyPress(CAMERA_CONTROL_KEY_ENTER, 0);
```

Comment:
I think the old one needs to be removed, we don't have any other function triggered by two different combinations.